src/apps/btc/btc_common.c

```diff
@@ -15,11 +15,15 @@
 #include <stdio.h>
 
 #include "btc_common.h"
+#include <apps/common/bip32.h>
 
 #include <apps/common/bip32.h>
+#include <keystore.h>
 #include <util.h>
 #include <wally_address.h>
 
+#define MULTISIG_P2WSH_MAX_SIGNERS (sizeof(multisig->xpubs) / sizeof(XPub))
```

Comment:
This seems like a complicated way to write 15. Maybe add a comment?

Reply:
Right, was too complicated. `15` works fine! Changed.